src/lib/support/DefaultStorageKeyAllocator.h

```diff
@@ -47,20 +54,23 @@ class DefaultStorageKeyAllocator
 
     // Access Control List
 
-    const char * AccessControlList() { return Format("acl"); }
+    // TODO: We should probably store the fabric-specific parts of the ACL list
+    // under keys starting with "f/%x/".
+    const char * AccessControlList() { return Format("g/acl"); }
```

Comment:
I have no particular problems with this, but... I'm going to likely change them again anyways, when I get a crack at improving the persistence for ACL, so if it's a hassle, maybe no big deal to leave these ones until then.

Reply:
Yeah, not planning to touch that stuff for now.